functions/validator.js

```diff
@@ -1,6 +1,6 @@
 const GBFS = require('gbfs-validator')
 
-exports.handler = function(event, context, callback) {
+exports.handler = function (event, context, callback) {
```

Comment:
suggestion: I know that this is an existing function. However, it will be nice to have documentation for it.